diff --git a/setup.py b/setup.py
index cd7ae8a..013878a 100644
--- a/setup.py
+++ b/setup.py
@@ -23,8 +23,8 @@ setup( packages=[],
     #                     'python-nemo >=3.8',
     #                     'meld'],
     data_files   = [
-        ('/usr/share/nemo-python/extensions', ['src/nemo-compare.py']),
-        ('/usr/share/nemo-compare', ['src/nemo-compare-preferences.py', 'src/utils.py']),
-        ('/usr/bin', ['src/nemo-compare-preferences'])
+        ('share/nemo-python/extensions', ['src/nemo-compare.py']),
+        ('share/nemo-compare', ['src/nemo-compare-preferences.py', 'src/utils.py']),
+        ('bin', ['src/nemo-compare-preferences'])
     ]
 )
diff --git a/src/nemo-compare-preferences b/src/nemo-compare-preferences
index 8d3b672..3ad4d26 100755
--- a/src/nemo-compare-preferences
+++ b/src/nemo-compare-preferences
@@ -3,4 +3,4 @@
 import subprocess
 import sys
 
-sys.exit(subprocess.call(["/usr/share/nemo-compare/nemo-compare-preferences.py"]))
+sys.exit(subprocess.call(["@nemo-compare@/share/nemo-compare/nemo-compare-preferences.py"]))
diff --git a/src/nemo-compare.py b/src/nemo-compare.py
index ff2cf10..888e148 100644
--- a/src/nemo-compare.py
+++ b/src/nemo-compare.py
@@ -32,7 +32,7 @@ import gi
 gi.require_version('Nemo', '3.0')
 from gi.repository import Nemo, GObject, Gio
 
-sys.path.append("/usr/share/nemo-compare")
+sys.path.append("@nemo-compare@/share/nemo-compare")
 
 import utils
 
diff --git a/src/utils.py b/src/utils.py
index 004382b..742f267 100644
--- a/src/utils.py
+++ b/src/utils.py
@@ -38,7 +38,7 @@ PREDEFINED_ENGINES = ['meld', 'kdiff3', 'diffuse', 'kompare', 'fldiff', 'tkdiff'
 DEFAULT_DIFF_ENGINE = "meld"
 
 # where comparator engines are sought
-COMPARATOR_PATHS = ['/usr/bin', '/usr/local/bin']
+COMPARATOR_PATHS = ['/usr/bin', '/usr/local/bin', '@nemo-compare@']
 
 class NemoCompareConfig:
 
